testing/ef-tests/tests/executor.rs

```diff
@@ -75,11 +72,11 @@ fn executor(
         let env = ExecutorEnv::builder()
             .session_limit(None)
             .segment_limit_po2(SEGMENT_LIMIT_PO2)
-            .add_input(&to_vec(&chain_spec).unwrap())
-            .add_input(&to_vec(&input).unwrap())
+            .write_slice(&to_vec(&chain_spec).unwrap())
+            .write_slice(&to_vec(&input).unwrap())
```

Comment:
This can be:
```
.write(&chain_spec)
.write(&input)
```

Reply:
I considered this, but then you still need to unwrap and formatted it gets
```rust
.write(&chain_spec)
.unwrap()
.write(&input)
.unwrap()
```
which I felt was a bit harder to read. But I don't have a strong opinion on this 🤷

Reply:
If there's no strong need to go either way, I'd default to having our examples and demos to be consistent (and for future host/guest I/O explainer docs).